src/controller/RacingCarController.js

```diff
@@ -0,0 +1,50 @@
+import { OUTPUT } from '../constant.js';
+import Validator from '../Validator.js';
+
+export default class RacingCarController {
+  #racingCarService;
+  #inputView;
+  #outputView;
+
+  constructor(racingCarService, inputView, outputView) {
+    this.#racingCarService = racingCarService;
+    this.#inputView = inputView;
+    this.#outputView = outputView;
+  }
+
+  async run() {
+    await this.inputCarNames();
+    const tryNumber = await this.inputTryNumber();
```

Comment:
자동차 이름은 변수에 저장이 안되는데 시도 횟수는 변수에 저장되는 점이 통일 성이 떨어지는 것 같아요. 그리고 이해하기도 어렵구요.
`inputCarNames`라는 메서드가 `input`을 받아와 `Validate`를 하고 `this.#racingCarService.saveCars`에 인자로 전달되는데, `inputCarNames`라는 네이밍을 단독으로 봤을 때는 그 기능들이 떠오르지 않는 것 같아 아쉬워요. 
더군다가 시도 횟수를 받는 `inputTryNumber`에는 `return`값이 있는데 `inputCarNames`에는 없고, 하지만 둘의 네이밍은 거의 같아서 더 헷갈리는 것 같아요.

메서드들의 네이밍에 맞는 추상화 계층의 통일성, 함수 기능의 통일성을 가독성 측면에서 더 고려해보시면 좋을 것 같다는 의견입니다 ☺️

Reply:
그리고 현재 `tryNumber`가 `const`로, `printAdvancedResult`에 파라미터로 전달되는데 `tryNumber`를 프로퍼티로 가져가는 건 어떻게 생각하시는지 궁금합니다!
```suggestion
    await this.inputCarNames();
    this.#tryNumber = await this.inputTryNumber();
```

Reply:
가독성을 신경쓴다고 했는데... 제 편의를 위해 가독성을 해치게 되었군요..!

지예님 말씀에 동의합니다! 통일성이 지켜지지 않아 가독성이 떨이지고, 메서드명이 적절하지 않은 것 같아요!

지예님 덕분에 다시 한번 저의 코드 작성하는 법을 정리하게 된 거 같습니다! 감사합니당

지역 변수 선언 vs 필드 선언인데 저는 해당 메서드 안에서만 사용되는 변수라 지역 변수로 선언했습니다!
다른 메서드도 사용해야 되는 변수면 필드로 선언했을 거 같습니당